apps/server/src/modules/authentication/services/authentication.service.spec.ts

```diff
@@ -114,18 +114,13 @@ describe('AuthenticationService', () => {
 	describe('generateJwt', () => {
 		describe('when generating new jwt', () => {
 			it('should pass the correct parameters', async () => {
-				const mockCurrentUser: ICurrentUser = {
-					accountId: 'mockAccountId',
-					roles: ['student'],
-					schoolId: 'mockSchoolId',
-					userId: 'mockUserId',
-					isExternalUser: false,
-				};
-				await authenticationService.generateJwt(mockCurrentUser);
+				const mockCurrentStudentUser = iCurrentUserFactory.withRoleStudent().build();
+
+				await authenticationService.generateJwt(mockCurrentStudentUser);
 				expect(jwtService.sign).toBeCalledWith(
-					mockCurrentUser,
+					mockCurrentStudentUser,
 					expect.objectContaining({
-						subject: mockCurrentUser.accountId,
+						subject: mockCurrentStudentUser.accountId,
 					})
 				);
 			});
```

Comment:
the test case is about testig the generating of JWT for a random user not a specific role (student). 
```suggestion
			it('should pass the correct parameters', async () => {
				const mockCurrentUser = iCurrentUserFactory.withRole("random role").build();

				await authenticationService.generateJwt(mockCurrentUser);
				expect(jwtService.sign).toBeCalledWith(
					mockCurrentUser,
					expect.objectContaining({
						subject: mockCurrentUser.accountId,
					})
				);
			});
```

Reply:
the test was with student user before so i only use the factory here